spec/models/campaign_spec.rb

```diff
@@ -120,4 +120,28 @@
       expect(campaign.last_contribution).to eq(Time.current + 1.day)
     end
   end
+
+  context 'with seller distributor pairs' do
+    let!(:campaign) do
+      create(:campaign, :with_sellers_distributors)
+    end
+
+    it 'gets seller distributor pairs' do
+      csds = campaign.campaigns_sellers_distributors
+      pairs = csds.map do |csd|
+        distributor = csd.distributor
+        seller = csd.seller
```

Comment:
is this used anywhere?

Reply:
neither of the these variables are actually. just removed them